examples/great3/cgc.yaml

```diff
@@ -122,16 +132,32 @@ gal:
             x_max: 0.05
         beta: { type: Random }
 
+    # Specify a minimum flux fraction in each postage stamp.
+    # The idea is that if the galaxy is drawn into the postage stamp of size stamp_size (see image
+    # field) then the galaxy gets chucked out and a new one is selected.  This will depend not only
+    # on the galaxy parameters but the PSF as well, since the cut is imposed on the PSF-convolved
+    # object.
+    min_flux_frac: 0.99
+
+    # Also use a custom module to apply other GREAT3 cuts: SNR and resolution.
+    reject:
+        type: Great3Reject
+
 eval_variables:
-    # I don't understand this bit at all.  
+    # We use the fwhm of the PSF in a couple of other places.  To make sure the config parser
+    # knows it's a float variable, we put it here with the f code.
+    fpsf_fwhm: { type: Current, key : 'psf.items.0.fwhm' }
+
     # Here is the line in the original great3 scripts:
     # https://github.com/barnabytprowe/great3-public/blob/master/great3sims/noise.py#L97
     # The point was to have a consistent lower limit on the S/N level of the galaxies for
-    # different seeing sizes.  When the seeing is larger, a given flux galaxy will have 
-    # lower S/N.  To compensate, they added less noise in those cases.  However, the
-    # exact formula they used is still a bit of a mystery to me.
-    # They did this as a LookupTable, but the formula below is pretty close.
-    fmax_var: '$0.004 * math.exp(2.9 * (1.1-@psf.items.0.fwhm) * (0.95-@psf.items.0.fwhm))'
+    # different seeing sizes.  When the seeing is larger, a given flux galaxy will have
+    # lower S/N.  To compensate, they added less noise in those cases.  The formula that was used
+    # was based on running experiments with different seeing values, and checking how the S/N varied
+    # at fixed noise variance.  The results of those experiments were used to create a LookupTable
+    # in the GREAT3 scripts, but the formula below is pretty close.
+    fmax_var: '$0.004 * math.exp(2.9 * (1.1-psf_fwhm) * (0.95-psf_fwhm))'
```

Comment:
It's not live yet.  But yes.  Here: https://github.com/GalSim-developers/GalSim/wiki/Config-Values#shorthand-notation